code-review.qmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Code review"
+editor: source
+---
+
+## Code review goals
+
+Code review helps maintain software quality. It can also be a great way to share knowledge and create common standards and approaches across the group.
```

Comment:
one sentence per line makes for easier diffs.  If you merge main, this is set as default in _quarto.yml and just opening this doc, making a minor edit, and hitting "save" should do this for you.

Reply:
Done!